src/figuresClasses.ts

```diff
@@ -1,19 +1,74 @@
-export interface Figure {
 
+export interface Figure {
+  shape:string;
+  color:string;
+  getArea:Function;
```

Comment:
better immediately typed function
```suggestion
export interface Figure {
  shape: string;
  color: string;
  getArea(): number;
```